src/models/accommodation.js

```diff
@@ -47,7 +47,7 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false,
     },
     accommodationType: {
-      type: DataTypes.ARRAY(DataTypes.STRING),
+      type: DataTypes.STRING,
```

Comment:
Why did you changed  the data type?

Reply:
I realized it should be a single type... depending on the type the vendor wants to provide